janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -961,4 +964,20 @@ private AdminMask getAdminInterface() {
             throw new JanusGraphException(e);
         }
     }
+
+    /**
+     * Similar to {@link Function}, except that the {@code apply} method is allowed
+     * to throw {@link BackendException}.
+     */
+    private static interface BackendFunction<F, T> {
```

Comment:
I did not add this... It must have happened during a rebase but I am not sure why it would be showing up in this PR... For now I will leave and will look into this later